ui/raidboss/data/05-shb/raid/e9s.js

```diff
@@ -1,8 +1,384 @@
+        if (data.me === matches.target)
+          return output.tankBusterOnYou();
+
+        if (data.role === 'tank')
+          return output.tankSwap();
+
+        if (data.role === 'healer')
+          return output.tankBusters({ player: data.ShortName(matches.target) });
+      },
+      infoText: function(data, _, output) {
+        if (data.role !== 'tank' && data.role !== 'healer')
+          return output.avoidLaser();
+      },
+      outputStrings: {
+        tankBusterOnYou: {
+          en: 'Tank Buster on YOU',
+          de: 'Tank buster auf DIR',
+          fr: 'Tank buster sur VOUS',
+          ja: '自分にタンクバスター',
+          cn: '死刑点名',
+          ko: '탱버 대상자',
+        },
+        tankBusters: {
+          en: 'Tank Busters',
+          de: 'Tank buster',
+          fr: 'Tank buster',
+          ja: 'タンクバスター',
+          cn: '死刑',
+          ko: '탱버',
+        },
+        tankSwap: {
+          en: 'Tank Swap!',
+          de: 'Tankwechsel!',
+          fr: 'Tank swap !',
+          ja: 'タンクスイッチ',
+          cn: '换T！',
+          ko: '탱 교대',
+        },
+        avoidLaser: {
+          en: 'Avoid Laser',
+          de: 'Laser ausweichen',
+          fr: 'Évitez le laser',
+          ja: 'アバランチに避け',
+          cn: '躲避击退激光',
+          ko: '레이저 피하기',
+        },
+      },
+    },
+    {
+      id: 'E9S Obscure Woods',
+      netRegex: NetRegexes.startsUsing({ id: '55EE', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9S Flood Of Obscurity',
+      netRegex: NetRegexes.startsUsing({ id: '5907', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 3,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Place Bramble',
+        },
+      },
+    },
+    {
+      id: 'E9S Waste Away',
+      netRegex: NetRegexes.startsUsing({ id: '5617', source: 'Cloud Of Darkness', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'E9S Rejuvenating Balm',
+      netRegex: NetRegexes.startsUsing({ id: '5618', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away From Tethered Walls',
+        },
+      },
+    },
+    {
+      id: 'E9S Stygian Break Tether',
+      netRegex: NetRegexes.headMarker({ id: '00D4' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: 3,
+      response: Responses.breakChains(),
+    },
+    {
+      id: 'E9S Anti-Air Phaser Unlimited Out',
+      netRegex: NetRegexes.startsUsing({ id: '561[23]', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 1,
+      durationSeconds: 6,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go Tethered Wall Then Out',
```

Comment:
What about `Out -> Stacks -> Sides` and `Out -> Tank Spread -> Sides`, depending on roll?